Jenkinsfile

```diff
@@ -607,6 +607,7 @@ def testGo(prefix, packagesToTest) {
         'github.com/keybase/client/go/kbfs/libfuse': [
           flags: '',
           timeout: '10m',
+          citogo_extra : '--pause 2s'
```

Comment:
This only needs to be .25s I think, at least on my local machine.  But I guess this should be fine too, will just make CI take longer (but maybe is better at avoiding flakes?).

Reply:
i was concerned that CI is crappier than your machine.